content/en/armory-enterprise/quickspin/install-quick-spin.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Install Quick Spin 
+linkTitle: "Install Quick Spin"
+no_list: true
+weight: 4
+description: >
+  Use the Armory Quick Spin container to quickly get an evaluation or testing instance of Spinnaker up and running.
+exclude_search: true
+---
+
+## {{% heading "prereq" %}}
+
+
+To run the Quick `quick-spin` container you need a hosting machine that can support a running instance with 
+  - 4 vCPU
+  - 8GB Memory
+  - 2GB Disk space
+
+The hosting machine must have [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed. Docker Compose is installed with Docker Desktop.
+
+## Usage
+
+To run a local instance of `quick-spin` using [docker compose](https://docs.docker.com/compose/):
+
+1. Clone the [Quick Spin repository](https://github.com/armory-io/quick-spin).
+   
+```shell
+git clone https://github.com/armory-io/quick-spin.git
+```
+2. From the terminal navigate to the root of the cloned repository.
```

Comment:
```suggestion
1. Navigate to the root of the cloned repository.
```
Assumption is that user is already in the terminal since she already cloned the repo.

Reply:
Also, if you use `1.` to start each point, Hugo compiles into the correct number so you don't have to renumber if you expand or contract the list items.